snuba/protobufs/base_messages.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+message RequestInfo {
+  repeated uint64 project_ids = 1;
+  repeated uint64 organization_ids = 2;
+  string cogs_category = 3;
+  string referrer = 4;
+  google.protobuf.Timestamp start_timestamp = 5;
+  google.protobuf.Timestamp end_timestamp = 6;
+}
+
+enum Comparison {
+  EQ = 0;
+  NEQ = 1;
+  LE = 2;
+  GT = 3;
+  LTE = 4;
+  GTE = 5;
+  LIKE = 6;
+  NOT_LIKE = 7;
+  IN = 8;
+  NOT_IN = 9;
+}
+
+
+message PentityFilter{
```

Comment:
At a certain point this will just be a Protobuf version of the objects we have in the Snuba SDK, right?

Reply:
More restrictive. For example we won't allow arbitrary functions here like we do in the snuba SDK

Reply:
I'm actually going to remove these filters and use the ones Colin proposed

https://github.com/getsentry/snuba/blob/master/snuba/protobufs/Filters.proto